app/models/match_decisions/base.rb

```diff
@@ -86,7 +86,7 @@ def contact_name
     def editable?
       # can this decision be updated by a notification response?
       # override this default behavior in subclasses
-      initialized? && match_open? && saved_status !~ /\A(accepted|confirmed|declined|canceled|rejected|complete|completed|scheduled|no_hearing|mitigation_required|mitigation_not_required|decline_overridden|decline_overridden_returned|decline_confirmed)\z/
+      initialized? && match_open? && saved_status !~ /\A(accepted|acknowledged|confirmed|declined|canceled|rejected|complete|completed|scheduled|no_hearing|mitigation_required|mitigation_not_required|decline_overridden|decline_overridden_returned|decline_confirmed)\z/
```

Comment:
@dtgreiner why this change?  I feel like we removed `acknowledged` because it was causing problems once deployed.

Reply:
Thats right, i do remember that one. I added it here because of a display bug after the first step was acknowledged. It wasn't showing the completed checkmark in the breadcrumbs.
![Screenshot 2024-10-11 at 8 15 30 AM](https://github.com/user-attachments/assets/1909c2c6-b7b4-4b5b-8b28-d518bed63e1a)

I'll pull it out of here as this is less disruptive than the problems it was causing earlier. I can create a ticket to find a good resolution for the two issues.

Reply:
[Ticket added here](https://github.com/open-path/Green-River/issues/6835)